static/app/views/performance/cache/referrers.ts

```diff
@@ -1,6 +1,4 @@
-const PREFIX = `api.performance.cache`;
-
 export enum Referrer {
-  LANDING_CACHE_HIT_MISS_CHART = `${PREFIX}.landing-cache-hit-miss-chart`,
-  LANDING_CACHE_THROUGHPUT_CHART = `${PREFIX}.landing-cache-throughput-chart`,
+  LANDING_CACHE_HIT_MISS_CHART = `api.performance.cache.landing-cache-hit-miss-chart`,
+  LANDING_CACHE_THROUGHPUT_CHART = `api.performance.cache.landing-cache-throughput-chart`,
 }
```

Comment:
stuff like this could probably be a const enum or just an `as const` object

Reply:
I raise your recommendation: I think we should just avoid having any `enum` and use JS objects to reduce typescript execution duration.